src/main/java/com/exasol/adapter/dynamodb/mapping/ValueMapperException.java

```diff
@@ -0,0 +1,30 @@
+package com.exasol.adapter.dynamodb.mapping;
+
+import com.exasol.adapter.AdapterException;
+
+/**
+ * Exception on failures in column mapping
+ */
+public class ValueMapperException extends AdapterException {
```

Comment:
You have quite a lot of custom exceptions. Do you really need them?

Reply:
well yes. They contain hold additional information about the current state when the exception was fired